src/fideslang/models.py

```diff
@@ -278,10 +284,11 @@ class DataSubjectRights(BaseModel):
         description="Defines the strategy used when mapping data rights to a data subject.",
     )
     values: Optional[List[DataSubjectRightsEnum]] = Field(
+        default=None,
         description="A list of valid data subject rights to be used when applying data rights to a data subject via a strategy.",
     )
 
-    @root_validator()
+    @model_validator(mode="before")
```

Comment:
This was later updated to mode="after".   Generally, mode="after" was a lot friendlier the more I worked through changes, since values could be an arbitrary object otherwise.